client/my-sites/plugins/marketplace/components/index.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { HorizontalRule } from '@wordpress/components';
+import styled from '@emotion/styled';
+import { MarketplaceThemeType } from 'calypso/my-sites/plugins/marketplace';
+
+export const MobileHiddenHorizontalRule = styled( HorizontalRule )< {
+	theme?: MarketplaceThemeType;
+} >`
+	@media ( ${ ( { theme } ) => theme.breakpoints.tabletDown } ) {
```

Comment:
If `theme` is optional, should we ensure that it exists before accessing its properties?